open_academy/views/course.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.actions.act_window" id="action_list_ideas">
```

Comment:
`id` comes first, also rename the id with a with a name compatible with the guidelines: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id24
```suggestion
    <record  id="course_action" model="ir.actions.act_window">
```